hack/local-up-cluster.sh

```diff
@@ -858,7 +858,7 @@ function start_kubelet {
       done
     fi
     # Quick check that kubelet is running.
-    if [ -n "$KUBELET_PID" ] && ps -p $KUBELET_PID > /dev/null; then
+    if [ -n "$KUBELET_PID" ] && ps -p ${KUBELET_PID} > /dev/null; then
```

Comment:
missed one here?

Reply:
Whoops! Forgotten a few.
I've added them. But I try not to make the PR to big as it is more error prone when having to many changes at once.

Reply:
yeah, an iterative approach seems like a good idea. I mostly nitted here because it was on a line you already changed :)